nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+  stringData:
+    username: ${NUXEO_CONNECT_USERNAME}
+    password: ${NUXEO_CONNECT_PASSWORD}
+
+- apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/ssh-auth
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-github-ssh
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    ssh-privatekey: |-
+      ${SSH_PRIVATEKEY}
+
+- apiVersion: v1
+  kind: Secret
+  type: Opaque
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-instance-clid
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    instance.clid: |-
+      ${INSTANCE_CLID}
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}-nuxeo
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    dockerImageRepository: nuxeo
+
+- apiVersion: v1
+  data:
+    MAVEN_SETTINGS: |-
+      <?xml version='1.0' encoding='UTF-8'?>
+      <settings>
+        <servers>
+          <server>
+            <id>nuxeo-studio</id>
+            <username>${env.NUXEO_CONNECT_USERNAME}</username>
+            <password>${env.NUXEO_CONNECT_PASSWORD}</password>
+          </server>
+          <server>
+            <id>external-releases</id>
+            <username>${env.NEXUS_USERNAME}</username>
+            <password>${env.NEXUS_PASSWORD}</password>
+          </server>
+        </servers>
+        <profiles>
+          <profile>
+            <id>release</id>
+            <properties>
+              <external.releases.url>${env.EXTERNAL_RELEASES_URL}</external.releases.url>
+            </properties>
+          </profile>
+        </profiles>
+      </settings>
+  kind: ConfigMap
+  metadata:
+    name: ${APPLICATION_NAME}-maven-settings
+
+
+############################################################################################################
+# DEV Deployment
+############################################################################################################
+
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ${APPLICATION_NAME}-dev-conf
+    labels:
+      app: ${APPLICATION_NAME}-dev
+      role: config
+      stage: dev
+  data:
+    nuxeo.conf: |
+      # Additional nuxeo.conf parameters
+    init.sh: |
+      #!/bin/sh
+
+      ## Need to use nss_wrapper in order to make LibreOffice conversion works
+      ## cf https://docs.openshift.com/enterprise/3.2/creating_images/guidelines.html
+      export USER_ID=$(id -u)
+      export GROUP_ID=$(id -g)
+
+      cat > /tmp/passwd <<EOT
+      root:x:0:0:root:/root:/bin/bash
+      nuxeo:x:${USER_ID}:${GROUP_ID}:Nuxeo server:/tmp:/bin/bash
+      EOT
+
+      export LD_PRELOAD=/usr/lib/libnss_wrapper.so
+      export NSS_WRAPPER_PASSWD=/tmp/passwd
+      export NSS_WRAPPER_GROUP=/etc/group
```

Comment:
The NSS hack is not needed anymore with recent Docker images. Lines from 1031 to 1043 can be removed.

Reply:
removed